doc/tutorials/basic/serve.md

```diff
@@ -0,0 +1,232 @@
+# Build Hello World App
+
+In this tutorial, we will develop and *serve* a *Hello World* Panel:
+
+- Serve the app with `panel serve app.py`, `panel serve app.ipynb`, or `panel serve app.md`.
+- serve with auto reload by adding the flag `--autoreload`.
+- stop your server with `CTRL+C`.
+
+## Serve the App
+
+Select a *tab* below and continue
+
+:::::{tab-set}
+
+::::{tab-item} Python Script
+:sync: script
+
+The simplest Panel `.py` app could look like this:
+
+```python
+import panel as pn
+
+pn.extension()
+
+pn.panel("Hello World").servable()
+```
+
+:::{note}
+The code refers to
+
+- `panel`: The Panel python package. It's a convention to import it as `pn`.
+- `pn.extension()`: Loads javascript dependencies and configures Panel.
+- `pn.panel(...)`: Creates a *displayable* Panel component.
+- `.servable()`: Displays the component in a *server app*.
+:::
+
+Copy the code into a file named `app.py`.
+
+Save the file.
+
+Run the Panel server in your terminal with
+
+```bash
+panel serve app.py --autoreload
+```
+
+When the server has started, the terminal output should look like this
+
+```bash
+$ panel serve app.py --autoreload
+2024-01-17 15:49:11,443 Starting Bokeh server version 3.3.2 (running on Tornado 6.3.3)
+2024-01-17 15:49:11,444 User authentication hooks NOT provided (default user enabled)
+2024-01-17 15:49:11,450 Bokeh app running at: http://localhost:5006/app
+2024-01-17 15:49:11,450 Starting Bokeh server with process id: 47256
+```
+
+:::{note}
```

Comment:
Maybe.

I put it in a note because that is what the much appraised FastAPI docs does.

![image](https://github.com/holoviz/panel/assets/42288570/c0ff639f-6312-4eb5-82d5-0d90e8925394)

FastAPI docs uses *note, *hint*, *check* *admonitions* all over the place. I have done the same. I think they are useful because they help break up the page. But exactly when and how to use them is more an art than a science.

Push me harder or get other contributors to also suggest changing this then I will do it.

Reply:
> FastAPI docs uses note, hint, check admonitions all over the place. I have done the same.

You've mostly used `note` admonitions, right?

> But exactly when and how to use them is more an art than a science.

True! I think overall in the current Panel tutorials `note` are overly-used.